test-integration/configs/validator-offline.devnet.toml

```diff
@@ -21,7 +21,7 @@ max-snapshots = 7
 snapshot-frequency = 1024
 
 [ledger]
-resume-strategy = { variant = "reset" }
+resume-strategy = { kind = "reset" }
```

Comment:
Not sure if I missed it somewhere, but does `resume-strategy = "reset"` also still work in this case?
Not absolutely needed, but would be nice.